#75 Adding view plan button

pinkdragon1000 opened the pull request.

pinkdragon1000 committed:
#75 Adding view plan button

pinkdragon1000 committed:
adding more default logic

msd96drexel reviewed (changes requested):
When I jump sections, it still says 'view plan' on the button

pinkdragon1000 committed:
fixing issue

Tried to simplify logic but it was only working before hitting previous.  So changing it back.

msd96drexel commented on frontend/src/components/page_content/page-content-template.js:
```diff
@@ -32,6 +32,10 @@ class PageContentTemplate extends React.Component {
     this.prev = this.prev.bind(this);
   }
 
+  save() {
```
if you're adding the button you should also make it link to the plan page

pinkdragon1000 committed:
Adding link to plan page on view plan click

pinkdragon1000 committed:
fixing issue

pinkdragon1000 committed:
Merge branch 'master' into viewplan

msd96drexel commented on frontend/src/components/page_content/page-content-template.js:
```diff
@@ -30,6 +30,14 @@ class PageContentTemplate extends React.Component {
+  }
+
+  save() {
+    //Save form and move to view plan
```
don't need this comment - we can add the save later when we go to implement it

msd96drexel commented on frontend/src/components/page_content/page-content-template.js:
```diff
@@ -45,17 +53,23 @@ class PageContentTemplate extends React.Component {
   }
 
   setFactorsSaveCancel() {
+    this.saveLabel = "Next";
```
i think this should be set in the constructor cuz it always has to render as something, so you can be sure its set by putting it there instead

msd96drexel commented on frontend/src/components/page_content/page-content-template.js:
```diff
@@ -45,17 +53,23 @@ class PageContentTemplate extends React.Component {
+      this.saveLabel = "View Plan";
+      this.saveClick = this.save;
+    } else {
+      this.saveLabel = "Next";
```
you won't need this else part if the saveLabel should be set in the constructor

msd96drexel commented on frontend/src/components/page_content/page-content-template.js:
```diff
@@ -30,6 +30,14 @@ class PageContentTemplate extends React.Component {
+
+  save() {
+    //Save form and move to view plan
+    if (this.saveLabel === "View Plan") {
```
i think you can just put this if statement in "next()" and then set the onSave={this.next}. then you can have else increment step. you shouldn't need save() or this.saveClick

pinkdragon1000 committed:
Updates based on comments in pr #148, Issue #75

pinkdragon1000 committed:
Updates based on comments in pr #148 , Issue #75
